app/src/directives/context-menu/context-menu.ts

```diff
@@ -0,0 +1,40 @@
+import { Directive, DirectiveBinding } from 'vue';
+
+function mounted(element: HTMLElement, binding: DirectiveBinding): void {
+	const contextMenu = binding.instance?.$refs['contextMenu'];
```

Comment:
I don't think we can make this automated 🤔 There's gonna be cases where theres's more than one context menu in the same parent component. I think we should update this so the contextMenu ref is pulled in from the binding, which then means the use becomes:

`<element v-context-menu="contextMenuRef">`

Reply:
> There's gonna be cases where theres's more than one context menu in the same parent component. 

That's true 😄 Updated the PR with this change.